<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerConfiguration</class>
 <widget class="QWidget" name="WorkerConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Worker Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../ressources/ressources.qrc">
    <normaloff>:/img/icons/effector</normaloff>:/img/icons/effector</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Working batch configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Population size</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of generations</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Selection weight</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>From results bests</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>From results random</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>From active pop bests</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>From active pop random</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>From random seed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="sldBestResults">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="sldRandomResults">
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="sldBestPop">
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="sldRandomPop">
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="sldRandomSeed">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Number of bests results stored</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Number of random results stored</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Launch</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Worker name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtWorkerName">
          <property name="text">
           <string>Anonymous</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="spbPopSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="spbMaxGen">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="spbBestResults">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QSpinBox" name="spbRandomResults">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="pbReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../ressources/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sldBestResults</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_10</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldRandomResults</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_11</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldBestPop</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_12</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldRandomPop</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_13</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldRandomSeed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_14</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
